src/Core/Silk.NET.Core/Native/SilkMarshal.cs

```diff
@@ -451,6 +462,7 @@ public static GlobalMemory PtrToMemory(IntPtr ptr, int length)
         /// </remarks>
         /// <param name="memory">The memory to zero-initialize the start of.</param>
         /// <param name="memoryLength">The length of the memory.</param>
+        [Obsolete("Random method added in the 2.0 preview cycle, to be removed any day now.")]
```

Comment:
just remove it imo. It's already removed from SilkTouch

Reply:
That PR is still pending, so not yet.

Reply:
PR updated, maybe we can get that in before this?

Reply:
Done, will just let CI run through.

Reply:
CI doesn't indicate any problem.